src/server/serverless/lambda-migrate-user-links/index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+const path = require('path')
+const { Client } = require('pg')
+
+const SQLSCRIPT = 'migrate_user_links.sql'
+
+async function handler(event) {
+  const dbConfig = process.env.DATABASE_URL
+  const pgClient = new Client(dbConfig)
+  let statusMsg
+
+  try {
+    const { fromUserEmail } = event
+    const { toUserEmail } = event
+    const sqlScriptPath = path.join(__dirname, SQLSCRIPT)
+
+    await pgClient.connect().then(() => {
+      console.log('Connected')
+    })
+
+    const sqlUDF = fs.readFileSync(sqlScriptPath)
+    const sqlScript = `SELECT migrate_user_links($1, $2)`
+    const values = [fromUserEmail, toUserEmail]
+
+    const queryUDFResult = await pgClient.query(sqlUDF.toString())
```

Comment:
I'm still not very comfortable of the idea of overwriting the function every single time we want to invoke it. Is there no way to avoid this at all?

Reply:
Thanks for pointing it out, didn't realise it's already stored in the DB - have removed the lines pertaining to the SQL function